packages/core-ethereum/docs/modules/contracts_hoprchannels.md

```diff
@@ -0,0 +1,13 @@
+[@hoprnet/hopr-core-ethereum](../README.md) / [Exports](../modules.md) / contracts/HoprChannels
+
+# Module: contracts/HoprChannels
```

Comment:
It could make sense to define sth like a maximum depth for which typedocs are created